systemd/kibana.service

```diff
@@ -0,0 +1,41 @@
+[Unit]
+Description=Kibana 4
+BindsTo=probemanager.service
+Wants=elasticsearch.service
+After=elasticsearch.service
+Wants=probemanager.service
+Before=probemanager.service
+Wants=probelogger.service
+Before=probelogger.service
```

Comment:
OK. If I remember once I'm on CentOS7. I wanted to point it out since it was one of the lessons learned from Cyan team (KISS)